nni/tools/nnictl/nnictl.py

```diff
@@ -212,26 +212,24 @@ def parse_args():
     parser_log_trial.add_argument('--trial_id', '-T', dest='trial_id', help='find trial log path by id')
     parser_log_trial.set_defaults(func=log_trial)
 
-    #parse package command
-    parser_package = subparsers.add_parser('package', help='control nni tuner and assessor packages')
-    # add subparsers for parser_package
-    parser_package_subparsers = parser_package.add_subparsers()
-    parser_package_install = parser_package_subparsers.add_parser('install', help='install packages')
-    parser_package_install.add_argument('source', nargs='?', help='installation source, can be a directory or whl file')
-    parser_package_install.add_argument('--name', '-n', dest='name', help='package name to be installed', required=False)
-    parser_package_install.set_defaults(func=package_install)
+    #parse algo command
+    parser_algo = subparsers.add_parser('algo', help='control nni builtin tuner, assessor and advisor algorithms')
```

Comment:
suggest to do some backward compatibility. for example, we keep `nnictl package` for several releases, and changes its implementation to simply print message "`nnictl package` command is deprecated, please use `nnictl algo` instead"

Reply:
It would be great to be backward compatible.  But considering we need to maintain 2 sets of meta formats / interfaces to implement it.  How about tell user `nnictl package command is replaced by nnictl algo command`  when user trying to use `nnictl package` and provide better help info for `nnictl algo`?

Reply:
agree, providing a message is enough

Reply:
Thanks, updated.